app/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "network": "development",
+    "testnet": {
+        "services": "0x60b1a811f5ea71806bb3bc0e5b74920eeae91913",
+        "endpoint": "http://?.?.?.?:8545"
```

Comment:
@gnidan I've made this change now. See app.js I'm pulling client host from truffle.js. I had to remove the default builder from truffle.js as I get errors on import in app.js. I think it doesn't matter and the default builder gets set anyway? In any case i'm using webpack seperatly to build. Let me know if it all looks okay.